src/main/scala/firrtl/Emitter.scala

```diff
@@ -62,6 +62,17 @@ class VerilogEmitter extends Emitter with PassBased {
       s".${name}($strx)"
     case RawStringParam(name, value) => s".$name($value)"
   }
+  def stringify(tpe: GroundType): String = tpe match {
+    case (_: UIntType | _: SIntType | _: AnalogType) =>
+      val wx = bitWidth(tpe) - 1
+      if (wx > 0) s"[$wx:0]" else ""
+    case ClockType => ""
+    case _ => error("Shouldn't be here")
```

Comment:
Make this more descriptive... "Trying to write an unsupported type in the Emitter"